Add `OmitIndexSignature` and `PickIndexSignature` types (#453)
sindresorhus/type-fest: readme.md

- [`RequireAtLeastOne`](source/require-at-least-one.d.ts) - Create a type that requires at least one of the given keys.
- [`RequireExactlyOne`](source/require-exactly-one.d.ts) - Create a type that requires exactly a single key of the given keys and disallows more.
- [`RequireAllOrNone`](source/require-all-or-none.d.ts) - Create a type that requires all of the given keys or none of the given keys.

> - [`OmitIndexSignature`](source/omit-index-signature.d.ts) - Omit any index signatures from the given object type leaving any explicitly defined properties.

Reviewer: ```suggestion
- [`OmitIndexSignature`](source/omit-index-signature.d.ts) - Omit any index signatures from the given object type, leaving only explicitly defined properties.
```

> - [`PickIndexSignature`](source/pick-index-signature.d.ts) - Pick any index signatures from the given object type removing any explicitly defined properties.

Reviewer: ```suggestion
- [`PickIndexSignature`](source/pick-index-signature.d.ts) - Pick only index signatures from the given object type, leaving out all explicitly defined properties.
```

Same with the description in the file.
